src/main/resources/com/nike/cerberus/migration/V1.6.2.0__add_type_to_secure_data.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE SECURE_DATA
+  ADD COLUMN `TYPE` VARCHAR(255) NOT NULL,
+  ADD COLUMN SIZE_IN_BYTES INTEGER NOT NULL;
+
+UPDATE SECURE_DATA
+  SET `TYPE` = 'OBJECT',
+      SIZE_IN_BYTES = -1;
```

Comment:
@fieldju I want to call this out directly and see if you have any objections/suggestions for default size value